From 9968ecf3ba0d9ba7afaa2c12e38d49fd6de05bc2 Mon Sep 17 00:00:00 2001
From: Lu Yang A <yang.a.lu@intel.com>
Date: Tue, 1 Nov 2022 15:32:10 +0800
Subject: [PATCH] Remove libmd.so from additional native libraries

Remove libmd.so from additional native libraries for CTS tests as
libmd is disabled.
Ported from master PR(OAM-104325)
https://github.com/projectceladon/device-androidia-mixins/pull/1511

Tracked-On: OAM-104242
Signed-off-by: Lu Yang A <yang.a.lu@intel.com>
Signed-off-by: Vasoya,Nikhilx <nikhilx.vasoya@intel.com>
---
 groups/public-libraries/true/public.libraries.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/groups/public-libraries/true/public.libraries.txt b/groups/public-libraries/true/public.libraries.txt
index d7fc8f2..28209b9 100644
--- a/groups/public-libraries/true/public.libraries.txt
+++ b/groups/public-libraries/true/public.libraries.txt
@@ -1,3 +1,2 @@
 # Specify additional native libraries
 # So that Apps can use Intel proprietary libraries
-libmd.so
-- 
2.38.1

